components/script/layout_wrapper.rs

```diff
@@ -908,6 +913,13 @@ impl<'ln> ThreadSafeLayoutNode for ServoThreadSafeLayoutNode<'ln> {
             self.get_jsmanaged().downcast::<Element>().unwrap().get_colspan()
         }
     }
+
+    #[inline]
+    fn quirks_mode(&self) -> QuirksMode {
```

Comment:
yeah, I think you're right. If we ever need it later, we can just add it at that time.